Gruntfile.js

```diff
@@ -63,16 +83,28 @@ module.exports = function(grunt) {
         watch: {
             js: {
                 files: ['js/**/*.js'],
-                tasks: ['concurrent', 'concat', 'uglify']
+                tasks: ['jslint', 'concat', 'uglify']
+            },
+            css: {
+                files: ['css/**/*.scss'],
+                tasks: ['scsslint', 'sass']
             }
         },
 
         concurrent: {
-            lint: ['jshint', 'jscs']
+            jslint: ['jslint'],
+            js: [['concurrent:jslint', 'concat', 'uglify']],
```

Comment:
I think I saw an example where by adding the double parenthesis it runs the tasks in order - seeing as I don't want the `concat` task to run until the `jslint` task finishes, and I don't want to run the `uglify` task to run until the `concat` task finishes.